employee.rb

```diff
@@ -0,0 +1,22 @@
+
+class Employee
+  attr_accessor :name, :salary, :performance_review, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @performance_review = []
```

Comment:
You're setting these to arrays here, when I'm guessing that you're not going to put multiple things into them in the future.  I'd probably suggest deleting these two lines (since that's going to by default set them both to nil).  That will cause less confusion in readers later, since they're not going to start thinking that they should << into these variables rather than =.